src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -37,119 +62,212 @@ const FacilitySearch = props => {
         onChange: setSelectedFacilities,
         facilities,
         query: submittedQuery,
+        error: facilitiesListError,
       };
     },
-    [facilities, submittedQuery, props],
+    [
+      facilities,
+      submittedQuery,
+      props,
+      dispatch,
+      formData,
+      facilitiesListError,
+    ],
   );
 
   const handleChange = e => {
     setQuery(e.target.value);
   };
 
-  const handleClick = async () => {
+  const handleSearch = async () => {
     if (!query.trim()) {
-      setError(content['validation-facilities--search-required']);
+      setSearchInputError(content['validation-facilities--search-required']);
       return;
     }
 
     setLoading(true);
-    setError(null);
+    setFacilitiesListError(null);
+    setSearchInputError(null);
     setFacilities([]);
 
     const mapboxResponse = await fetchMapBoxGeocoding(query);
     if (mapboxResponse.errorMessage) {
-      setError(mapboxResponse.errorMessage);
+      setSearchInputError(mapboxResponse.errorMessage);
       setLoading(false);
       return;
     }
 
     const [longitude, latitude] = mapboxResponse.center;
+    setCoordinates({ long: longitude, lat: latitude });
+
     const facilitiesResponse = await fetchFacilities({
       long: longitude,
       lat: latitude,
     });
+
     if (facilitiesResponse.errorMessage) {
-      setError(facilitiesResponse.errorMessage);
+      setSearchInputError(facilitiesResponse.errorMessage);
       setLoading(false);
       return;
     }
 
     setFacilities(facilitiesResponse);
     setSubmittedQuery(query);
     setLoading(false);
+    setPages(1);
+    focusElement('#caregiver_facility_results');
+  };
+
+  const showMoreFacilities = async e => {
+    e.preventDefault();
+    setLoadingMoreFacilities(true);
+    const facilitiesResponse = await fetchFacilities({
+      ...coordinates,
+      page: pages + 1,
+      perPage: 5,
+    });
+
+    if (facilitiesResponse.errorMessage) {
+      setSearchInputError(facilitiesResponse.errorMessage);
+      setLoadingMoreFacilities(false);
+      return;
+    }
+
+    setFacilities([...facilities, ...facilitiesResponse]);
+    setSubmittedQuery(query);
+    setLoadingMoreFacilities(false);
+    setPages(pages + 1);
+  };
+
+  const loader = () => {
+    return (
+      <va-loading-indicator
+        label={content['app-loading-generic-text']}
+        message={content['facilities-loading-text']}
+        set-focus
+      />
+    );
   };
 
   const searchResults = () => {
     if (loading) {
+      return loader();
+    }
+    if (hasFacilities()) {
       return (
-        <va-loading-indicator
-          label={content['app-loading-generic-text']}
-          message={content['facilities-loading-text']}
-          set-focus
-        />
+        <>
+          <FacilityList {...facilityListProps} />
+          {loadingMoreFacilities && loader()}
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+          <a href="#" onClick={showMoreFacilities}>
```

Comment:
We need to refactor this to a `button`. This is an a11y violation. I assume you want it to look like a link? If so, we have a class for that. Lets do this as:
```
<button type="button" className="va-button-link" onClick={showMoreFacilities}>
    Load more facilities
</button>
```